spark-jobs/src/main/resources/application.conf.template

```diff
@@ -42,6 +42,10 @@ conformance.catalyst.workaround=true
 # Automatically delete standardized data folder after successful run of a Conformance job
 conformance.autoclean.standardized.hdfs.folder=false
 
+# Enforce info file validation
+# Can be one of: strict, warning, off (anything else will be treated as off)
+control.info.validation=warning
```

Comment:
```suggestion
standardized.control.info.validation=warning
```

```suggestion
standardization.control.info.validation=warning
```

If this is for standardization only, maybe the key could reflect that somehow?

Reply:
Actually it should have applied for conformance too, good point. I will use it in conformance too